antora.yml

```diff
@@ -8,6 +8,7 @@ asciidoc:
     javadocVersion: 7.13
     crowdinBonitaVersion: 7.13
     # page attributes
+    page-pagination: true # display links to previous/next pages at the bottom of the page
```

Comment:
Is it available to be displayed at the top of the page as well?

Reply:
Not with the current implementation, but it should be technically possible to add it in the future.
@NathalieC what do you think about it? I don't plan to do any change in the theme for now i.e. pagination will be for now only at the bottom or won't be added.

Reply:
I think it's an interesting feature, although not sure people actually need it besides the Getting Started pages.
But if it holds no usability risk at all, let's go for it. A few questions to check that aspect:

- How does it display with very long page names for both previous and next pages? 
- How does it handle the transition between two modules?
- How does it handle the transition between the last page of a chapter and the index page of the next chapter?

Plus, it may help us realize our page names are not that great and need so improvements (like "Bonita" in the example you have provided :/)
 "it is possible to apply the changes to the whole documentation site" --> Yeah, I think this is what we should do.

Reply:
okay, I've checked my 3 use-cases through the preview; it behaves great.
So, let's go for it :)